### ISSUE TITLE
[Feature request] Text Component

### ISSUE THREAD

**Issue by alex-zhang** [SAP/ui5-webcomponents#1858]
hi, 

we have completed this component in react way. but we thinks it's a basic component for too many situations. so, it will helpful for us which is supported by ui5 web component.

![image](https://user-images.githubusercontent.com/4457451/85397860-e4608f80-b586-11ea-9e38-2b159d0705d7.png)

fiori-design: https://experience.sap.com/fiori-design-web/text
ui5: https://sapui5.hana.ondemand.com/#/entity/sap.m.Text

**Comment by vladitasev** [comment]
Hi @alex-zhang 

Is there something in particular that you need, because as far as I can see from first glance, it's just a CSS class, defining `font-family`, `color`, `font-size` and maybe at most also wrapping behavior. Then you apply this CSS class to all texts.

It would be:
`<span class="fiori-text">Some text</span>`

compared to:
`<ui5-text>Some text</span>`

We have talked about this in the past and we decided to do `<ui5-title>` which was also very simple, but we thought `ui5-text` would be too little to require a component of its own.

Regards,
Vladi

**Comment by winterkind** [comment]
Hi @vladitasev 

I found this old feature request because I was wondering about a simple Text component, too. What you write in your reply makes sense, the styling for such a component would be very simple. It is easy to do that in my application.

Unless I want to support theming.

For example, take the [Page component in the playground](https://sap.github.io/ui5-webcomponents/playground/components/Page/). Switch to "Quartz dark" and you get dark text on dark background. This sample does not do anything to support dark themes.

The [Panel component in the playground](https://sap.github.io/ui5-webcomponents/playground/components/Panel/) is more sophisticated. It defines a class that supports "Quartz dark". But when you set the theme to "Quartz high contrast dark" the text is unreadable again because those playground styles do not support that theme.

In short: of course I can build an app that does what the Panel playground does and define classes with styles for different themes. Pick the right colors or CSS variables and keep them up-to-date.

But it would be helpful to have a simple Text component that does the theming for all supported themes for me.
